base/setup/usetup/usetup.c

```diff
@@ -3238,19 +3238,25 @@ CheckFileSystemPage(PINPUT_RECORD Ir)
 }
 
 
-static VOID
+static NTSTATUS
```

Comment:
Now of course you'd also need to check this functions return value... :p
(To be clear, I'm okay with not doing that in this PR)